src/utilities.js

```diff
@@ -14,15 +30,9 @@ const getLanguageCode = () => {
 };
 
 const createHasViewedCookieBanner = () => {
-  const host = window.location.hostname;
-
-  // edx.org uses different subdomains so use a root domain to match across services
-  if (host.indexOf('stage.edx.org') !== -1 ||
-      host.indexOf('dev.edx.org') !== -1 ||
-      host.indexOf('acceptance.edx.org') !== -1 ||
-      host.indexOf('qa.edx.org') !== -1) {
+  if (isStage()) {
```

Comment:
You could consider doing something like

```javascript
const createHasViewedCookie = () => {
  const domain = isProduction() ? '.edx.org' : '.stage.edx.org';
  return new Cookie().set('edx-cookie-policy-viewed', true, { domain });
};
```

Reply:
This isn't actually the same because I'm limiting what stage is (i.e. it needs to be one of the predefined environments). Let's discuss in stand-up whether we need to be this prescriptive, or if we are simply defining ```stage``` as ```!production```.

Reply:
FYI it looks like the domain for language preference on `extra` is actually `.edx.org`? Even though the cookie name is `stage-edx-language-preference`

![image](https://user-images.githubusercontent.com/8136030/38627601-5d8e44f8-3d7d-11e8-8441-63955b44120a.png)

I got here by removing both `prod|stage-edx-language-preference` and then going to `/es`.